src/core/exceptions.c

```diff
@@ -331,6 +331,13 @@ static LocatedHandler search_for_handler_from(MVMThreadContext *tc, MVMFrame *f,
  * an exception object already, it will be used; NULL can be passed if there
  * is not one, meaning it will be created if needed (based on the category
  * parameter; if ex_obj is passed, the category is not used). */
+static void cleanup_control_handler(MVMThreadContext *tc) {
```

Comment:
Is this guaranteed to be always called? What happens, when some other exception is thrown while the unwind is still in progress?